site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -36,110 +36,22 @@ See [Prerequisites](prerequisite-helm.md#Software-requirements) for more informa
-🔥  Creating docker container (CPUs=2, Memory=8100MB) ...
-❗  This container is having trouble accessing https://k8s.gcr.io
-💡  To pull new external images, you may need to configure a proxy: https://minikube.sigs.k8s.io/docs/reference/networking/proxy/
-🐳  Preparing Kubernetes v1.20.7 on Docker 20.10.7…
-    ▪ Generating certificates and keys ...
-    ▪ Booting up control plane ...
-    ▪ Configuring RBAC rules ...
-🔎  Verifying Kubernetes components...
-    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
-🌟  Enabled addons: storage-provisioner, default-storageclass
-🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
-```
-
-#### 3. Check the K8s cluster status
-
-Run `$ kubectl cluster-info` to check the status of the K8s cluster you just created. Ensure that you can access the K8s cluster via `kubectl`. You should see the following expected output.
-
-```
-Kubernetes control plane is running at https://127.0.0.1:63754
-KubeDNS is running at https://127.0.0.1:63754/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
-
-To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
-```
-
-### Create a K8s cluster with kind
-
-[kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container “nodes”.
-
-#### 1. Create a configuration file 
-
-Create the `kind.yaml` configuration file as follows.
-
-```
-kind: Cluster
-apiVersion: kind.x-k8s.io/v1alpha4
-nodes:
-- role: control-plane
-- role: worker
-- role: worker
-- role: worker
-```
-
-#### 2. Create a K8s cluster
-
-Create a K8s cluster using the `kind.yaml` configuration file.
-
-```
-$ kind create cluster --name myk8s --config kind.yaml
-```
-
-After the K8s cluster starts, you can see the following output. 
-
-```
-Creating cluster "myk8s" ...
- ✓ Ensuring node image (kindest/node:v1.21.1) 🖼
- ✓ Preparing nodes 📦 📦 📦 📦
- ✓ Writing configuration 📜
- ✓ Starting control-plane 🕹️
- ✓ Installing CNI 🔌
- ✓ Installing StorageClass 💾
- ✓ Joining worker nodes 🚜
-Set kubectl context to "kind-myk8s"
-You can now use your cluster with:
-
-kubectl cluster-info --context kind-myk8s
-
-Not sure what to do next? 😅  Check out https://kind.sigs.k8s.io/docs/user/quick-start/
-```
-
 #### 3. Check the K8s cluster status
 
-Run `$ kubectl cluster-info` to check the status of the K8s cluster you just created. Ensure that you can access the K8s cluster via `kubectl`. You should see the following expected output.
-
-```
-Kubernetes control plane is running at https://127.0.0.1:55668
-CoreDNS is running at https://127.0.0.1:55668/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
-
-To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
-```
+Run `$ kubectl cluster-info` to check the status of the K8s cluster you just created. Ensure that you can access the K8s cluster via `kubectl`. 
```

Comment:
Run `$ kubectl cluster-info` to check the status of the K8s cluster you just created. Ensure that you can access the K8s cluster via `kubectl`. If you have not installed `kubectl` locally, see [Use kubectl inside minikube](https://minikube.sigs.k8s.io/docs/handbook/kubectl/).

Reply:
[916e971](https://github.com/milvus-io/milvus-docs/pull/1643/commits/916e9716eab8fc4f41361c064845d4c2c25d73c2)